sql/moz-fx-data-shared-prod/stripe_external/subscriptions_changelog_v1/query.sql

```diff
@@ -1,3 +1,7 @@
+{% if is_init() %}
+INSERT INTO
+ `{project_id}.{dataset_id}.{table_id}`
+{% endif %}
```

Comment:
Per my other comment I don't think this `INSERT INTO` approach is necessary, but if we do go with this approach then IMO the `INSERT INTO {table}` part should either be added dynamically by the tooling at runtime when needed or only added to queries that are actually using parallelized init. Not be added to every query that just happens to use the `is_init()` approach to init.

It's also worth noting this would change the current init behavior of overwriting the table contents and instead always append to the table.

Reply:
Indeed, it was work in progress to validate the best approach. I removed the need to modify existing queries and how they're currently updated. For running in parallel the sample_id range can be added to the metadata.